### ISSUE TITLE
pre release not useable

### ISSUE THREAD

**Issue by kanink007** [nullptrspace/DWS#6]
hello
i tried to test the DWS Extended Pre-Alpha release. but there are many dll-files missing. i downloaded them and in the end there was this error: 

"__vcrt_cleanup_type_info_names" not found in "vcruntime140d.dll"

how do i solve this error? i have no idea.

**Comment by Daniil1288** [comment]
found here:
https://twitter.com/nummerok/status/726603297537298433
translated to:
"visual studio 2015 must be installed"

**Comment by dartraiden** [comment]
Yes, seems like debug version compiled, so you need VS installed or manually download and put this libraries.
